chain-signatures/node/src/protocol/signature.rs

```diff
@@ -165,7 +149,7 @@ pub struct SignatureGenerator {
     pub presignature_id: PresignatureId,
     pub request: ContractSignRequest,
     pub epsilon: Scalar,
-    pub receipt_id: CryptoHash,
+    pub receipt_id: [u8; 32],
```

Comment:
why change these to non-`CryptoHash` type?

Reply:
ok, so talked with @volovyks about this one and it seems like we're moving to a more generic naming for these because other chains don't have this. We can change it to some other type later

Reply:
Have you tried using the newly introduced `SignRequestIdentifier` everywhere?